src/js/tech/tech.js

```diff
@@ -339,13 +395,13 @@ class Tech extends Component {
    */
 
   textTracks() {
-    this.player_.textTracks_ = this.player_.textTracks_ || new TextTrackList();
-    return this.player_.textTracks_;
+    this.textTracks_ = this.textTracks_ || new TextTrackList();
```

Comment:
Also, we don't actually want to shift it to the tech. This is because if you switch techs, you would lose the text tracks. We do not want that to happen.
There's probably something extra need to do to make text tracks work with techs that have their own tracks like the youtube tech, but for regular old tracks we want it on the player.